bootstrapvz/providers/gce/__init__.py

```diff
@@ -42,6 +42,13 @@ def resolve_tasks(taskset, manifest):
                     ])
     taskset.discard(grub.SetGrubConsolOutputDeviceToSerial)
 
+    # Temporary fix for Stretch
+    if manifest.release == testing:
```

Comment:
You specify`stretch` in the comment, so you should use `boostrapvz.common.releases.stretch`. This ensures forwards compatibility once stretch becomes `stable` (regardless of how temporary this is).